src/models/donation-info/donation-payment-info.ts

```diff
@@ -69,22 +69,6 @@ export class DonationPaymentInfo {
     return currencyValue;
   }
 
-  /**
-   * A default DonationInfo object
-   *
-   * @readonly
-   * @static
-   * @type {DonationPaymentInfo}
-   * @memberof DonationPaymentInfo
-   */
-  static get default(): DonationPaymentInfo {
```

Comment:
what happened to this?

Reply:
Removing this was one of those moments that came out of writing unit tests. I was trying to instantiate the donation form with an arbitrary testing donation amount, but for some reason, it kept defaulting to this default. I was using this default amount in a few places so they'd default to _something_ but that ended up having side effects where events would propagate with the wrong amount because they got initialized with it.

Now I [create the default](https://github.com/internetarchive/iaux-donation-form/pull/1/files/d0e3d6f6988190f2728822077f2015616f58b815..274ba3ca97520e595560f88d061d3fde49ac7a4f#diff-8f8897d9246680d2b2ecfc031a5cb63eR278-R282) in the donation form controller and it gets passed in to all of the child components that need it.

Reply:
the default at the controller is good - for quick reference.